src/main/java/com/pusher/client/channel/impl/message/EncryptedReceivedData.java

```diff
@@ -0,0 +1,17 @@
+package com.pusher.client.channel.impl.message;
+
+import com.pusher.client.util.internal.Base64;
+
+public class EncryptedReceivedData {
```

Comment:
I'm assuming this doesn't get exposed to the end user? Can it be made private/protected?